lib/puppet/provider/sqlserver_features/mssql.rb

```diff
@@ -127,7 +127,7 @@ def create
       instance_version = PuppetX::Sqlserver::ServerHelper.sql_version_from_install_source(@resource[:source])
       Puppet.debug("Installation source detected as version #{instance_version}") unless instance_version.nil?
 
-      install_net_35(@resource[:windows_feature_source]) unless [SQL_2016, SQL_2017, SQL_2019].include? instance_version
+      install_net_35(@resource[:windows_feature_source]) unless [SQL_2016, SQL_2017, SQL_2019, SQL_2022].include? instance_version
```

Comment:
What .net does this actually install? AFAIK from 2016 4.6 is the min requirement.

Reply:
Oh you may have picked up on something, looks like the dev box comes with/was installing .NET 4.7 so the installation was not failing.
But like you mentioned looks like .NET 4.6 is the minimum requirement for sql server 2016, I'll update this asap

Reply:
Thanks @chelnak  :-) So it looks like this ensures/installs .NET 3.5 for SQL Server 2012 + 2014. It seems, at least for the case of SQL Server 2012, that .NET is not automatically installed during SQL Server installtion like later versions (see [here](https://learn.microsoft.com/en-us/sql/sql-server/sql-server-2012-release-notes?view=sql-server-2016#:~:text=The%20.NET%20Framework%203.5%20SP1%20is%20a%20requirement%20for%20SQL%20Server%202012%20when%20you%20select%20Database%20Engine%2C%20Replication%2C%20Master%20Data%20Services%2C%20Reporting%20Services%2C%20Data%20Quality%20Services%20(DQS)%2C%20or%20SQL%20Server%20Management%20Studio%2C%20and%20the%20framework%20is%20no%20longer%20installed%20by%20SQL%20Server%20Setup.))

Reply:
Just more on this, think it makes more sense to have 
```
install_net_35(@resource[:windows_feature_source]) if [SQL_2012, SQL_2014].include? instance_version
```
instead so have updated